pygdf/series.py

```diff
@@ -409,6 +412,40 @@ def dtype(self):
         """dtype of the Series"""
         return self._impl.dtype
 
+    @classmethod
+    def _concat(cls, objs, index=True):
+        head = objs[0]
+        for o in objs:
+            if o._impl != head._impl:
+                raise ValueError("All series must be of same type")
+
+        data = head._impl.concat(objs)
```

Comment:
Fair point. I didn't realize `_data` also had a `dtype` attribute, I was relying on `self.dtype` (which could be `category`) when I implemented it before. Fixed.